nixos/doc/manual/release-notes/rl-2305.section.md

```diff
@@ -362,6 +362,8 @@ In addition to numerous new and upgraded packages, this release has the followin
 
 - A new `virtualisation.rosetta` module was added to allow running `x86_64` binaries through [Rosetta](https://developer.apple.com/documentation/apple-silicon/about-the-rosetta-translation-environment) inside virtualised NixOS guests on Apple silicon. This feature works by default with the [UTM](https://docs.getutm.app/) virtualisation [package](https://search.nixos.org/packages?channel=unstable&show=utm&from=0&size=1&sort=relevance&type=packages&query=utm).
 
+- New module `python3Packages.datetime`
```

Comment:
This is wrong, `python3Packages.datetime` is not a module but a package for which we usually do not write changelog entries

Reply:
What should I put instead?

Reply:
Or should I not put anything in the changelog?

Reply:
> Or should I not put anything in the changelog?

Yes. 
I didn't know this because I had never added it to the release note before, but I can't find a section about adding new packages in any of the past release notes.

Reply:
OK great I've removed from the changelog and force pushed an update. Let me know how it looks.